### ISSUE TITLE
Add an option to choose if hiden columns in list will be reported

### ISSUE THREAD

**Issue by nvdaes** [nvaccess/nvda#10403]
### Is your feature request related to a problem? Please describe.
In alpha versions, NVDA doesn't report hiden columns in lists (issue #9873). Now, when we navigate by columns with control+alt+right and left arrows, NVDA is still reporting the column number according to the original column order, and so the number of steps in table navigation can be inconsistent respect to the column number reported, if the list contains hiden columns.
### Describe the solution you'd like
A settings in Presentation object category to choose if hiden columns will be reported by NVDA, similar to screen layout option used for browse mode.
### Describe alternatives you've considered
This could be treated as a regression, but @leonardder and me have discussed this and we think it's not appropriate, as shown in [messages posted on the development mailing list](https://groups.io/g/nvda-devel/message/44433):

### Additional context
Sometimes, we may want to use the _getColumnNumber function. This method may not to be marked as private.

**Comment by Adriani90** [comment]
cc: @ABuffEr

**Comment by ABuffEr** [comment]
Maybe it could be useful to provide a str. I, and imagine @nvdaes too, see this behavior in eMule search results (enabling table cell coordinates reporting, that I keep usually disabled). In that situation, file name, that appears as first column, is reported as 10th column. I agree that it could be better...

**Comment by nvdaes** [comment]
Hello, at this moment, in search results of  eMule, I see name as the first columna when using controlalt
arrow keys, but it's considered the fourth column, so that it's reported by pressing NVDA+control+4 when using our eMule add-on. Hope this can be addressed.

**Comment by Adriani90** [comment]
I agree having an option in advanced NVDA settings would make sense. Blind developers might need the current behavior, but the normal user would rather prefer the expected behavior.
